### PR TITLE
feat: add runable demo microservices for fully development experience

### PR DESCRIPTION
**Pull request by kaufmo**

### CODE REVIEW

**Comment by nsams** [comment]
is this a 1:1 copy from gitlab?

**Comment by johnnyomair** [comment]
> is this a 1:1 copy from gitlab?

Yes! Although we made some changes for going open source:

1. The authentication library has been changed to `@comet/react-app-auth`
2. The certificates for local development are now generated using `mkcert` (view README)

File: demo/admin/intl-update.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { existsSync, promises, readFileSync } from "fs";
+import { homedir, hostname } from "os";
+import { dirname } from "path";
+
+let gitlabToken = process.env.GITLAB_API_TOKEN;
+if (!gitlabToken) {
+    const gitlabTokenFile = `${homedir()}/.gitlab_token`;
+    if (!existsSync(gitlabTokenFile)) {
+        console.warn(
+            `Environment-Variable GITLAB_API_TOKEN not present and ${gitlabTokenFile} doesn't exist on '${hostname()}'. Please create a personal token at https://gitlab.vivid-planet.com/profile/personal_access_tokens and save it into this file (both on your machine and your virtual machine on the server`,
```
Comment (dkarnutsch) [inline demo/admin/intl-update.ts:11]: This should probably not be here

**Review by dkarnutsch — CHANGES_REQUESTED** [review]

File: demo/admin/intl-update.ts (hunk 1 above)
Comment (johnnyomair) [reply demo/admin/intl-update.ts]: Fixed in d9372c21f78f7371dda4799bccf5898ff9d8dd25

**Review by dkarnutsch — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -5,6 +5,15 @@
 ### Requirements
 
 -   [nvm](https://github.com/nvm-sh/nvm)
+-   [mkcert](https://github.com/FiloSottile/mkcert)
+
+#### Install certificates once.
+
+```bash
+mkcert -install
+mkcert -key-file certs/privkey.pem -cert-file certs/cert.pem "*.comet-dxp.dev"
```
Comment (fraxachun) [inline README.md:14]: These files are not included, how is this meant?

**Review by fraxachun — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (johnnyomair) [reply README.md:14]: mkcert creates these files. The naming of the options is rather confusing, as it indicates that these files are used.

**Comment by kaufmo** [comment]
:partying_face:

**Pull request merged**